pkg/qiniu/qnPresubmit.go

```diff
@@ -107,20 +107,26 @@ func FindBaseProfileFromQiniu(qc *Client, prowJobName, covProfileName string) ([
 	return qc.ReadObject(profilePath)
 }
 
-// Artifacts prepresents the rule to store test artifacts in prow
-type Artifacts struct {
+type Artifacts interface {
```

Comment:
suggest to add comments for exported interface

Reply:
added~

Reply:
`therule` => `the rule`